7c60903edaf0efcc501c0f689c01bef2773947cc/COVID_FAQ.Rmd

```diff
@@ -0,0 +1,47 @@
+---
+title: "COVID_FAQ"
+author: "Crystal Nguyen"
+date: "11/2/2020"
+output: html_document
+---
+
+
+<br>
+
+**Postponement Updates**
+
+As of 10/6/2020, we have made the unfortunate and difficult decision to postpone our wedding. With no vaccine available, cases on the rise again, and flu season here, we cannot see a feasible way to keep ourselves, our loved ones (that's you!), and our vendors safe, especially with most of the guests and bridal party (including ourselves) traveling in from out of state .
+
+We apologize for any inconvenience this postponement may bring you. No one is more sorry than we are that we won't get to host our wedding on our originally intended date! That said, between this postponement and all the social distancing, we are all the more excited to celebrate with you when the time comes.
+
+As of 10/31/2020, we have been able to secure a new venue for 02/26/2022. We will update this website as soon as possible with the latest details as we work them out, so please stay tuned! Below are answers to some common questions about our postponement
```

Comment:
Do we need to say "we will update..."? We already have the new venue address. I guess finalizing the hotel block is still a detail to update?